neo/Network/P2P/Payloads/BlockBase.cs

```diff
@@ -50,18 +64,18 @@ void IVerifiable.DeserializeUnsigned(BinaryReader reader)
             NextConsensus = reader.ReadSerializable<UInt160>();
         }
 
-        UInt160 IVerifiable.GetScriptHashForVerification(Snapshot snapshot)
+        UInt160[] IVerifiable.GetScriptHashesForVerifying(Snapshot snapshot)
         {
-            if (PrevHash == UInt256.Zero) return Witness.ScriptHash;
+            if (PrevHash == UInt256.Zero) return new[] { Witness.ScriptHash };
             Header prev_header = snapshot.GetHeader(PrevHash);
             if (prev_header == null) throw new InvalidOperationException();
-            return prev_header.NextConsensus;
+            return new[] { prev_header.NextConsensus };
         }
 
         public virtual void Serialize(BinaryWriter writer)
         {
             ((IVerifiable)this).SerializeUnsigned(writer);
-            writer.Write(Witness);
+            writer.Write((byte)1); writer.Write(Witness);
```

Comment:
What is this one? Version? If it's version, or something else, better to keep on a named variable.

Reply:
https://github.com/neo-project/neo/blob/206876e48cbef4c6f6110b31ac0a5b71feb0b42a/neo/Network/P2P/Payloads/BlockBase.cs#L37-L48

Reply:
Oh man... I just got it.  :joy: Let's Serialize it inside a `Witness[]` then, it's much more clear...